### PR TITLE
Feature/search bar

### PR DESCRIPTION
**Pull request by REFAEI91**

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -23,12 +23,7 @@ const Title = styled.h2`
   text-shadow: 0 1px 2px rgba(0, 0, 0, 0.32);
   font-family: ${goldman.style.fontFamily};
 `;
-const Summary = styled.p`
-  font-size: 0.9rem;
-  margin-bottom: 10px;
-  text-shadow: 0 1px 2px rgba(0, 0, 0, 0.32);
-  word-wrap: break-word;
-`;
+const Summary = styled.p``;
```
Comment (Roland-Hufnagel) [inline components/Card.js]: If you don't style it you can use a normal `p`-tag.

File: components/CardList.js [hunk 2]
```diff
@@ -1,20 +1,70 @@
...
+`;
+const Results = styled.p`
+  text-align: center;
+`;
 export default function CardList() {
+  const [searchQuery, setSearchQuery] = useState("");
+  const handleInputChange = (event) => {
+    setSearchQuery(event.target.value);
+  };
+  console.log(searchQuery);
```
Comment (Roland-Hufnagel) [inline components/CardList.js]: please remove console.logs

File: components/CardList.js [hunk 3]
```diff
@@ -1,20 +1,70 @@
...
+  const filteredMedications = medications.filter(
+    (medication) =>
+      medication.name.toLowerCase().includes(searchQuery.toLowerCase()) ||
+      medication.summary.toLowerCase().includes(searchQuery.toLowerCase())
+  );
+  console.log(filteredMedications);
```
Comment (Roland-Hufnagel) [inline components/CardList.js]: here the same

File: components/CardList.js [hunk 4]
```diff
@@ -1,20 +1,70 @@
...
+            {filteredMedications.map((medication) => (
+              <li key={medication.id}>
+                <Card medication={medication} />
+              </li>
+            ))}
+          </>
+        ) : (
+          <Results>No results found</Results>
+        )}
+      </List>
```
Comment (Roland-Hufnagel) [inline components/CardList.js]:
```suggestion
        {filteredMedications.length > 0 ? (
          <List>
            {searchQuery && (
              <Results>
                {filteredMedications.length === 1
                  ? `1 result found`
                  : `${filteredMedications.length} results found `}
              </Results>
            )}
            {filteredMedications.map((medication) => (
              <li key={medication.id}>
                <Card medication={medication} />
              </li>
            ))}
          </List>
        ) : (
          <Results>No results found</Results>
        )}
```

Comment (Roland-Hufnagel) [reply components/CardList.js]: I suggest to render the list only if there is any content (list-items).

**Review by Roland-Hufnagel — APPROVED** [review]
This looks and works very fine. I just left some miner important suggestions...

**Review by Roland-Hufnagel — APPROVED** [review]
This looks and works very fine. I just left some miner important suggestions...

File: components/CardList.js (hunk 4 above)
Comment (REFAEI91) [reply components/CardList.js]: thank you for your helpful suggestion , I will do it now

**Pull request merged**